devito/core/gpu_openacc.py

```diff
@@ -74,6 +74,11 @@ def _map_present(cls, f):
         return cls.lang['map-present'](f.name, ''.join('[0:%s]' % i
                                                        for i in cls._map_data(f)))
 
+    @classmethod
+    def _map_delete(cls, f):
```

Comment:
note that `lang['map-release']` is mapped to `acc exit data delete` as well, just like the current `lang['map-exit-delete']`. This is my fault of course. I wonder whether we can clean up this mess...

Reply:
TBH, I'm not familiar with OpenACC yet. So I'd be happy to address that issue in a new PR with a small patch.